repo_resource/common.py

```diff
@@ -357,17 +364,21 @@ def update_manifest(self, jobs):
                 xml = ET.parse('.repo/manifests/'+self.__name)
                 manifest = xml.getroot()
 
-                # Get default values from manifest
```

Comment:
why moving this around? 

Can't we keep this block here and add the following at the end of the manifest parsing?

```
if defaultBranch is None:
    defaultBranch = self.__remote_rev(defaultRemote)
```

Reply:
We could, but it's neater to fill in remote_revision and then set the default value (order brings more logic) in a OR condition which is more straight forward IMHO

Reply:
Ok, you we can keep as is.